src/Microsoft.AspNet.Razor/Parser/TagHelpers/TagHelperBlockBuilder.cs

```diff
@@ -134,6 +134,6 @@ public override void Reset()
         /// <summary>
         /// The starting <see cref="SourceLocation"/> of the tag helper.
         /// </summary>
-        public SourceLocation Start { get; private set; }
+        public SourceLocation Start { get; set; }
```

Comment:
In this PR we need it accessible for testing yes. However, the builder was a bit odd to begin with only having a `private set`. All of our other builds have public get/set methods, thought I'd just normalize this property since I had to touch it anyways.